src/javascript/app/pages/bottom/graph.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+import { Skeleton } from '@deriv-com/quill-ui';
+
+const Graph = ({ renderGraph }) => {
+    const [isMounted, setMounted] = useState(false);
+
+    useEffect(() => {
+        const mountTimer = setTimeout(() => {
+            setMounted(true);
+            const renderTimer = setTimeout(() => {
+                const cleanupRender = renderGraph();
+
+                return cleanupRender;
+            }, 1000);
+
+            return () => clearTimeout(renderTimer);
```

Comment:
all this `return` statements inside `setTimeout`s allows us to call `mountTimer()` or `cleanupRender()` instead of the reqular`clearTimeout(<id>)` in order to clean up the timer. But you never use them like this.

If the goal is to clean up the timers, we don't need the `return` statement inside of the timeouts :)
Each `setTimeout` returns an `id` by default which we can use to clean it up by passing this `id` to `clearTimeout()` inside the `useEffect` cleanup (return statement).